src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -786,8 +797,38 @@ def create(self, validated_data):
             # Save updated information on the ZAAK
             zaak.save(update_fields=_zaak_fields_changed)
 
+            # update deelzaken
+            self.update_deelzaken(
+                zaak.deelzaken,
+                archiefactiedatum=zaak.archiefactiedatum,
+                startdatum_bewaartermijn=zaak.startdatum_bewaartermijn,
+            )
+
         return obj
 
+    def update_deelzaken(self, qs, **fields):
+        self._update_deelzaken_with_internal_catalogi(
+            qs.filter(_zaaktype__isnull=False), **fields
+        )
+        self._update_deelzaken_with_external_catalogi(
+            qs.filter(_zaaktype_relative_url__isnull=False), **fields
+        )
+
+    def _update_deelzaken_with_internal_catalogi(self, qs, **fields):
```

Comment:
if `archiefactiedatum` and `startdatum_bewaartermijn` are the only fields passed, let's make it explicit:

```suggestion
    def _update_deelzaken_with_internal_catalogi(self, qs, archiefactiedatum, startdatum_bewaartermijn):
```